ec2utils/ec2utils.conf.example

```diff
@@ -11,6 +11,7 @@ access_key_id =
 secret_access_key = 
 ssh_key_name =
 ssh_private_key =
+subnet_id_us-east-1 = subnet-123456
```

Comment:
Could subnet_id be in region section?

```
[region-us-east-1]
subnet_id = subnet-123456
```

Reply:
subnet is region and account specific, for example I upload with -a rschweikert and -a resale-images to us-east-1, the subnet cannot be shared between these accounts so it needs to be account specific, but also needs to be region specific because subnets do not span regions.

Concocting the config value name from "fixed-identifier" + region was from my point of view the most straight forward option otherwise the config file might have needed to be re-constructed, which would be more painful, IMHO